src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -271,11 +271,11 @@
     updateNamingExclusions: function(component, hh) {
         var strExclusions = '';
         if (!component.get('v.isAutoName'))
-            strExclusions += 'Household__c.Name;';
```

Comment:
Why this change? Haven't we always been using these weird strings? While I prefer to remove the Household__c portion, I'm worried about the possible side effects of changing this without a good mitigation strategy.